.gitignore

```diff
@@ -1,7 +1,5 @@
 node_modules
-server/public/bundle.js
-server/public/bundle.js.map
-server/public/bundle.js.LICENSE.txt
+server/public/**.js
```

Comment:
This doesn't seem to work. Looks like you've included bundle.js.map in this PR.

Reply:
fixed